src/api/ZoweExplorerApi.ts

```diff
@@ -488,5 +488,13 @@ export namespace ZoweExplorerApi {
          * @returns the registered API instance
          */
         getJesApi(profile: IProfileLoaded): IJes;
+
+        /**
+         * After an extenders registered all its API extensions it
+         * might want to request that profiles should get reloaded
+         * to make them automatically appears in the Explorer drop-
+         * down dialogs.
+         */
+        reloadProfiles?(): Promise<void>;
```

Comment:
This method should be part of the ZoweExplorerExtender class along with the getProfile and getLinkedProfile methods. https://github.com/zowe/vscode-extension-for-zowe/pull/705

Reply:
@phaumer @Colin-Stone I've created a new branch to try merging the two branches and moving the `reloadProfiles` function as requested: https://github.com/zowe/vscode-extension-for-zowe/tree/profile-link-w-refresh